rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @data = date
```

Comment:
[ ] OPTIONAL
I think you want to say @date here, instead of @data

Reply:
thank for noticing, i have resolved